concourse-ci/azuredeploy.json

```diff
@@ -113,7 +113,7 @@
     "ubuntuOSVersion": "14.04.3-LTS",
     "sshKeyPath": "[concat('/home/', parameters('adminUsername'), '/.ssh/authorized_keys')]",
 
-    "baseUriAzureCloud": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/concourse-ci/",
+    "baseUriAzureCloud": "https://raw.githubusercontent.com/cerdmann-pivotal/azure-quickstart-templates/concourse-update/concourse-ci/",
```

Comment:
@cerdmann-pivotal 
The best practice by azure-quickstart-templates is to add a parameter `_artifactsLocation`. For testing, you can specify the parameter with your private URL.
FYI:
https://github.com/Azure/azure-quickstart-templates/blob/master/bosh-setup/azuredeploy.json#L88-L95